lib/modules/datasource/index.ts

```diff
@@ -311,7 +311,8 @@ async function fetchReleases(
     config.additionalRegistryUrls,
   );
   let dep: ReleaseResult | null = null;
-  const registryStrategy = datasource.registryStrategy ?? 'hunt';
+  const registryStrategy =
+    config.strategy ?? datasource.registryStrategy ?? 'hunt';
```

Comment:
I think we should create a test for this

Reply:
yeah, where should I put it?

Reply:
ok